sim/warlock/doomguard.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 func (warlock *Warlock) registerSummonDoomguard(timer *core.Timer) {
-	duration := time.Duration(45+10*warlock.Talents.AncientGrimoire) * time.Second
+	duration := time.Duration(45+10*warlock.Talents.AncientGrimoire+warlock.Tier132PSummonDuration) * time.Second
```

Comment:
It would be nicer to make the tier bonus a helper function in items that calculates the summon duration extension instead of having a variable in the warlock struct that's only ever going to be written once.